doc/CONFIGURATION.md

```diff
@@ -896,66 +863,56 @@ const node = await createLibp2p({
+Some Transports can be passed additional options when they are created. For example, [webRTC](../packages/transport-webrtc) accepts optional [DataChannel Options](https://github.com/libp2p/js-libp2p/blob/master/packages/transport-webrtc/src/stream.ts#L13-L17). In addition to libp2p passing itself and an `Upgrader` to handle connection upgrading, libp2p will also pass the options, if they are provided, from `config.transport`.
 
 ```js
 import { createLibp2p } from 'libp2p'
-import { webRTCStar } from '@libp2p/webrtc-star'
-import { mplex } from '@libp2p/mplex'
 import { yamux } from '@chainsafe/libp2p-yamux'
 import { noise } from '@chainsafe/libp2p-noise'
-import wrtc from 'wrtc'
+import { webRTC } from '@libp2p/webrtc'
 
-const webRTC = webRTCStar({
-  wrtc
-})
 
 const node = await createLibp2p({
   transports: [
-    webRTC.transport
-  ],
-  peerDiscovery: [
-    webRTC.discovery
+    webRTC({
+      dataChannel: {
+        maxMessageSize: 10
+      }
+    })
   ],
   streamMuxers: [
-    yamux(),
-    mplex()
+    yamux()
   ],
   connectionEncryption: [
     noise()
   ]
 })
 ```
 
-During Libp2p startup, transport listeners will be created for the configured listen multiaddrs.  Some transports support custom listener options and you can set them using the `listenerOptions` in the transport configuration. For example, [libp2p-webrtc-star](https://github.com/libp2p/js-libp2p-webrtc-star) transport listener supports the configuration of its underlying [simple-peer](https://github.com/feross/simple-peer) ice server(STUN/TURN) config as follows:
+During Libp2p startup, transport listeners will be created for the configured listen multiaddrs.  Some transports support custom listener options and you can set them using the `listenerOptions` in the transport configuration. For example, [webRTC](../packages/transport-webrtc) transport listener supports the configuration of ice servers (STUN/TURN) config as follows:
 
 ```js
-const webRTC = webRTCStar({
-  listenerOptions: {
-    config: {
-      iceServers: [
-        {"urls": ["turn:YOUR.TURN.SERVER:3478"], "username": "YOUR.USER", "credential": "YOUR.PASSWORD"},
-        {"urls": ["stun:YOUR.STUN.SERVER:3478"], "username": "", "credential": ""}]
-    }
-  }
-})
-
 const node = await createLibp2p({
   transports: [
-    webRTC.transport
-  ],
-  peerDiscovery: [
-    webRTC.discovery
+    webRTC({
+      rtcConfiguration: {
+        iceServers:[{
+          urls: [
+            'stun:stun.l.google.com:19302',
+            'stun:global.stun.twilio.com:3478'
+          ]
+        }]
+    }
+})
   ],
   streamMuxers: [
-    yamux(),
-    mplex()
+    yamux()
   ],
   connectionEncryption: [
     noise()
   ],
   addresses: {
-    listen: ['/dns4/your-wrtc-star.pub/tcp/443/wss/p2p-webrtc-star'] // your webrtc dns multiaddr
+    listen: ['/dns4/your-wrtc-star.pub/tcp/443/wss/p2p-webrtc'] // your webrtc dns multiaddr
```

Comment:
To listen on available relay servers for incoming webrtc connections this should be:

```suggestion
    listen: ['/webrtc']
```

The relay transport should also be configured to search for relays.